extension/js/common/core/common.ts

```diff
@@ -159,7 +160,7 @@ export class Str {
     if (email.includes(' ')) {
       return false;
     }
-    email = email.replace(/\:8001$/, ''); // for MOCK tests, todo: remove from production
+    email = email.replace(new RegExp(`:${MOCK_PORT}$`), ''); // for MOCK tests, todo: remove from production
```

Comment:
is it still used or we can remove this replacement?

Reply:
I thought it is still used but let me check again

Reply:
Checked and this logic is still used